### PR TITLE
ChangedPileupDoc

### PR DESCRIPTION
**Pull request by chandrans**
Minor changes to doc.

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/qc/Pileup.java [hunk 1]
```diff
@@ -26,13 +26,13 @@
  *
  * <p>This tool emulates the 'samtools pileup' command. It prints the alignment in a format that is very similar to the
  * Samtools pileup format (see the <a href="http://samtools.sourceforge.net/pileup.shtml">Pileup format
- * documentation</a> for more details about the original format). There is one line per genomic position, listing the
+ * documentation</a> for more details about the original format). There is one column per genomic position, listing the
```
Comment (magicDGS) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/qc/Pileup.java]: I think that each line is a genomic position, and then each column shows the information for the list in the following line. I do not really know how to phrase it better, but the current one looks like each column is a genomic position and the first row is the chromosome name, the second the coordinate and so on (transposed format).

Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/qc/Pileup.java]: It would be great to include an example snippet of the output.

**Comment by sooheelee** [comment]
I find it funny we show the samtools command that our tool emulates....

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/qc/Pileup.java (hunk 1 above)
Comment (vdauwera) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/qc/Pileup.java]: there is an example snippet lower down in the file

Comment (vdauwera) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/qc/Pileup.java]: And yes it is one line per position, not one column. Fixed.

**Comment by vdauwera** [comment]
@droazen Not sure why we're getting a build failure here, can you or @lbergelson please take a look?

**Comment by vdauwera** [comment]
added missing documented feature annotation

**Comment by vdauwera** [comment]
@sooheelee Your comment about the missing doc feature annotation in another PR clued me in to the nature of the problem in this branch, thank you

**Pull request merged**